website/docs/r/schema_site_anp_epg_domain.html.markdown

```diff
@@ -35,35 +35,68 @@ resource "mso_schema_site_anp_epg_domain" "site_anp_epg_domain" {
   micro_seg_vlan           = 46
   port_encap_vlan_type     = "vlan"
   port_encap_vlan          = 45
-  enhanced_lag_policy_name = "Lacp"
-  enhanced_lag_policy_dn   = "uni/vmmp-VMware/dom-VMware-ab/vswitchpolcont/enlacplagp-Lacp"
 }
 
 ```
 
 ### domain_dn usage ###
 
 ```hcl
-resource "mso_schema_site_anp_epg_domain" "site_anp_epg_domain" {
-  schema_id = "5c4d9fca270000a101f8094a"
-  template_name = "Template1"
-  site_id = "5c7c95b25100008f01c1ee3c"
-  anp_name = "ANP"
-  epg_name = "Web"
-  domain_dn = "uni/vmmp-VMware/dom-VMware-ab"
-  deploy_immediacy = "lazy"
-  resolution_immediacy = "lazy"
-  vlan_encap_mode = "static"
-  allow_micro_segmentation = true
-  switching_mode = "native"
-  switch_type = "default"
-  micro_seg_vlan_type = "vlan"
-  micro_seg_vlan = 46
-  port_encap_vlan_type = "vlan"
-  port_encap_vlan = 45
+
```

Comment:
It might be good to add: ### domain_dn usage in version < 4.2 ###

Reply:
added also in examples